README.md

```diff
@@ -39,6 +39,11 @@ customization:
       officialExtensions: # optional
         - siderolabs/gvisor
         - siderolabs/amd-ucode
+overlay: # optional
+  name: rpi_generic # overlay name
+  image: ghcr.io/siderolabs/sbc-raspberry-pi # overlay image
```

Comment:
I wonder if we should swap these, as `image` defines an overlay, and `name` is one of the overlays in the image?

Reply:
didn't get this, `name` is the overlays in the image, not sure I follow

Reply:
I mean order of the fields